internal/features/cli/new/environment_delete.feature

```diff
@@ -0,0 +1,77 @@
+@environment @environment_delete
+Feature: Ernest environment delete
+
+  Scenario: User with project role deletes an environment
+		Given I setup ernest with target "https://ernest.local"
```

Comment:
Hey, @g3kk0 seems you're having some issues with your IDE formatting, please have a look and fix this :)